asttokens/mark_tokens.py

```diff
@@ -331,11 +381,13 @@ def handle_num(self, node, value, first_token, last_token):
     return (first_token, last_token)
 
   def visit_num(self, node, first_token, last_token):
+    # type: (NodeNG, util.Token, util.Token) -> Tuple[util.Token, util.Token]
     return self.handle_num(node, node.n, first_token, last_token)
 
   # In Astroid, the Num and Str nodes are replaced by Const.
   def visit_const(self, node, first_token, last_token):
-    if isinstance(node.value, numbers.Number):
```

Comment:
This is probably the source of the complex number test failure...

Reply:
And solved by c0bdb710fe790b73bcad3cef4629119fb075b989